src/django_components/templatetags/component_tags.py

```diff
@@ -560,6 +575,88 @@ def _parse_tag_body(parser: Parser, end_tag: str, inline: bool) -> NodeList:
     return body
 
 
+def _fix_nested_tags(parser: Parser, token: Token) -> None:
+    block_token = token
+
+    # When our template tag contains a nested tag, e.g.:
+    # `{% component 'test' "{% lorem var_a w %}"`
+    #
+    # Django parses this into:
+    # `TokenType.BLOCK: 'component 'test'     "{% lorem var_a w'`
+    #
+    # Above you can see that the token ends at the end of the NESTED tag,
+    # and includes `{%`. So that's what we use to identify if we need to fix
+    # nested tags or not.
+    has_unclosed_tag = block_token.contents.count("{%") > block_token.contents.count("%}")
```

Comment:
This wouldn't work for this crazy usage I think: `{% component 'test' "{% lorem var_a="{%}" %}`? Do we care? :)

Reply:
Of course we do! Fixed it :)